extensions/rich_text_components/Skillreview/directives/oppia-noninteractive-skillreview.directive.ts

```diff
@@ -82,6 +87,12 @@ angular.module('oppia').directive('oppiaNoninteractiveSkillreview', [
               }
             });
           };
+          ctrl.canOpenConceptCardModal = function(event) {
```

Comment:
```suggestion
          var canOpenConceptCardModal = function(event) {
```

Can we do this^? (We try to attach function/variable on ctrl only if we need to use them on templates.)

Reply:
Made the changes. Just as a question - do we only use the "this" reference within component files?